providers/github_test.go

```diff
@@ -31,7 +31,7 @@ func testGitHubBackend(payload []string) *httptest.Server {
 	pathToQueryMap := map[string][]string{
 		"/user":        {""},
 		"/user/emails": {""},
-		"/user/orgs":   {"limit=200&page=1", "limit=200&page=2", "limit=200&page=3"},
+		"/user/orgs":   {"per_page=200&page=1", "per_page=200&page=2", "per_page=200&page=3"},
```

Comment:
I am not really sure about these changes because I don't know how to run the tests. I assume the tests will be executed by travis CI? Any help appreciated ;)

Reply:
you can run these tests like so:

```
go test -v  github.com/pusher/oauth2_proxy/providers
```

I found that these params come in the other order now, "page" first.

I also found that doing `assert.Equal(t, nil, err)` was causing a panic, if you fix that to `assert.Nil(t, err)` you get a much nicer error message:

```
--- FAIL: TestGitHubProviderGetEmailAddressWithOrg (0.00s)
        Error Trace:    github_test.go:127
        Error:		Expected nil, but got: &errors.errorString{s:"got 404 from \"http://127.0.0.1:62568/user/orgs?page=1&per_page=100\" "}
        
        Error Trace:    github_test.go:128
        Error:		Not equal: "michael.bland@gsa.gov" (expected)
        	        != "" (actual)
```

leading to the appropriate test fix :)